eip/EIPS/eip-4337.md

```diff
@@ -147,7 +178,7 @@ function withdrawTo(address payable withdrawAddress, uint256 withdrawAmount) ext
 
 ```
 
-### Client behavior upon receiving a UserOperation
+### Bundler behavior upon receiving a UserOperation
```

Comment:
Why only Bundler?  The UserOp is checked by every client that participates in the mempool.  If Client is confusing then maybe "Bundler/Node"?

Reply:
removed. though I think we mixed some Bundler and Client in the text. but that's a separate PR .